### PR TITLE
CNCF Updates

### PR DESCRIPTION
**Pull request by ColinSullivan1**
* Apache 2.0 License
* Add code of conduct
* Add owners file
* Update copyright

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,22 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (aricart) [inline LICENSE]: Need to update this with "$BIRTH-2018 The NATS Authors"

File: OWNERS [hunk 2]
```diff
@@ -0,0 +1,10 @@
+reviewers:
+- aricart
```
Comment (aricart) [inline OWNERS]: ...

File: LICENSE (hunk 1 above)
Comment (ColinSullivan1) [reply LICENSE]: Not sure we do that in the license here.  These are boilerplate instructions...   An example elsewhere:  https://github.com/kubernetes/kubernetes/blob/master/LICENSE

Comment (ColinSullivan1) [reply LICENSE]: Let's keep as is for now, and key off of what we decide to do in gnatsd.

**Pull request merged**